bitrise.yml

```diff
@@ -51,7 +51,7 @@ workflows:
                 --data "client_id=bitrise-steps" \
                 --data "client_secret=$CACHE_API_CLIENT_SECRET" \
                 --data "grant_type=urn:ietf:params:oauth:grant-type:uma-ticket" \
-                --data "claim_token=eyJhcHBfaWQiOlsiY2FjaGUtc3RlcHMtdGVzdHMiXSwgIm9yZ19pZCI6WyJ0ZXN0LW9yZy1pZCJdLCAiYWJjc19hY2Nlc3NfZ3JhbnRlZCI6WyJ0cnVlIl19" \
+                --data "claim_token=eyJhcHBfaWQiOlsiMzYxNzJhODkyMTU1OTk1MSJdLCAib3JnX2lkIjpbIjg2NGMyZmViOTE0YzI2MTUiXSwgImFiY3NfYWNjZXNzX2dyYW50ZWQiOlsidHJ1ZSJdfQ==" \
```

Comment:
This is just a minor change: This base64 string used to contain a fake app and org ID, which now confuses the API regarding the cache quota. I changed this to use the real app and org ID of the `go-steputils` Bitrise project.